src/ranged.cpp

```diff
@@ -1475,14 +1473,17 @@ static bool is_driving( const player &p )
     return veh && veh->velocity != 0 && veh->player_in_control( p );
 }
 
-
 // utility functions for projectile_attack
-dispersion_sources player::get_weapon_dispersion( const item &obj, float range ) const
+dispersion_sources player::get_weapon_dispersion( const item &obj ) const
 {
-    dispersion_sources dispersion = obj.gun_dispersion();
-
+    int weapon_dispersion = std::max( obj.gun_dispersion() / 3, 45 );
```

Comment:
The division by three and minimum dispersion value are temporary fixes, they need to be applied across the weapon definitions.